### ISSUE TITLE
[v3.0.0-rc.22] Windows db-import issue with tab completion

### ISSUE THREAD

**Issue by uberhacker** [lando/lando#1823]
Yeah, I know...I should be using Linux instead of Windows but just thought I would check it out to see how well it works.  I noticed that tab completion in Windows PowerShell adds `.\` to the front of files but that creates a problem with `lando db-import .\backup.sql.gz`.  See below:
```
PS C:\Users\ed\projects\myproject> lando db-import .\backup-201910141357.sql.gz
File /app/.\backup-201910141357.sql.gz not found!
```
Removing the leading `.\` fixes the issue:
```
PS C:\Users\ed\projects\myproject> lando db-import backup-201910141357.sql.gz
Preparing to import /app/backup-201910141357.sql.gz into drupal8 on localhost:3306 as root...
Destroying all current tables in drupal8...
NOTE: See the --no-wipe flag to avoid this step!
Gzipped file detected!
Importing /app/backup-201910141357.sql.gz...

Import complete!
```
System specs:
```
Edition       Windows 10 Pro
Version       1903
Installed on  9/15/2019
OS build      18362.418
```
Docker Desktop:
```
Version  2.1.0.3 (38240)
Channel  stable
Engine   19.03.2
Compose  1.24.1
```

**Comment by uberhacker** [comment]
no @stalebot!

**Comment by Silic0nS0ldier** [comment]
If you can track down the appropriate source (I've had no luck), I'd guess that this could be fixed with some path normalisation.

**Comment by Silic0nS0ldier** [comment]
Unlike this has been resolved since my last message, but as before this shouldn't be a difficult problem to resolve. Just need to know where the change needs to be made.

**Comment by Silic0nS0ldier** [comment]
I had another look, and it would seem a bash script (`plugins\lando-services\scripts\sql-import.sh:47`) is responsible for handling the path. There likely isn't any reliable way to pre-process file paths, so this normalisation will probably need to be handled by the script itself.

This issue would also impact `db-export`, so there may be an opportunity to consolidate source common to both scripts (which a comment in `db-export` already notes).

**Comment by uberhacker** [comment]
Unstale.

**Comment by rabauss** [comment]
Use git bash on windows and that problem does not exist anymore ;-)

**Comment by Silic0nS0ldier** [comment]
Alternatively I can finally get around to making a PR for a potential fix I've had sitting on my machine for a few weeks. Only thing holding me back has been figuring out how to test it post-lando-integration.

**Comment by Silic0nS0ldier** [comment]
I've still to find a way to test the change once its integrated to lando, but I'm tired on sitting on it. PR has been opened with the changes.

**Comment by pirog** [comment]
I can replicate this using PowerShell and can also confirm it is not present using Git Bash as noted by @rabauss. Can also confirm that a slight variant on the solution by @Silic0nS0ldier in #2042 seems to do the trick

**Comment by pirog** [comment]
OK, i've merged #2042 with a tweak into `master` which means this should be in the next release!
